app/models.py

```
@@ -127,37 +128,54 @@ class Certification(Document):
-    DateTimeField, ReferenceField, IntField
-)
+    updated_at = DateTimeField(default=ist_now)
+
+    meta = {
+        'indexes': [
+            'meeting_id',
+            'date_time',
+            'status',
+            'employees'
+        ]
+    }
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)